app/controllers/admin/reports/registered_adviser_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Admin
+  module Reports
+    class RegisteredAdviserController < Admin::ApplicationController
+      respond_to :html, :csv
+
+      def index
+        @registered_advisers = Adviser.all.page(params[:page])
+        @registered_advisers_list = AdviserListCsv.new(Adviser.all)
```

Comment:
Can we pass on initialize, the @registered_advisers instead? Will be less hit on the DB.